clippy.toml

```diff
@@ -0,0 +1 @@
+too-many-arguments-threshold = 10
```

Comment:
are there a lot of functions that have more than the default number of arguments?